doc/catalog-format-spec.rst

```diff
@@ -285,6 +285,32 @@ entries:
    linux   = { OS = "gnu-linux" } # Compact table syntax is convenient in this case
    windows = { OS = "ms-linux" }  # to see all enumeration values, one per row.
 
+* ``environment``: optional table used to modify environment variables that
+  will apply at build time. Variables and values are specified with the form
+  ``VARIABLE.<action> = "value"``, where ``<action>`` is one of ``append``,
+  ``prepend``, or ``set``. For instance:
+
+  .. code-block:: toml
+
+   [environment]
+   C_INCLUDE_PATH.append = "/usr/include/something"
```

Comment:
Is the case maintained here? `PATH` vs `path`.

Reply:
Yes, TOML keys are case sensitive and I did no modification on these.